$MirOS: ports/www/firesomething/patches/patch-nsprpub_pr_src_md_unix_unix_errors_c,v 1.1 2006/05/11 20:55:44 bsiegert Exp $
--- nsprpub/pr/src/md/unix/unix_errors.c.orig	Sat Aug 23 17:04:11 2008
+++ nsprpub/pr/src/md/unix/unix_errors.c	Mon Aug  3 18:41:06 2009
@@ -212,11 +212,6 @@ void _MD_unix_map_default_error(int err)
         case EOPNOTSUPP:
             prError = PR_NOT_TCP_SOCKET_ERROR;
             break;
-#ifdef EOVERFLOW
-        case EOVERFLOW:
-            prError = PR_BUFFER_OVERFLOW_ERROR;
-            break;
-#endif
         case EPERM:
             prError = PR_NO_ACCESS_RIGHTS_ERROR;
             break;
